data/api/client-server/create_room.yaml

```
@@ -87,12 +87,9 @@ paths:
                     - public
                     - private
                   description: |-
-                    A `public` visibility indicates that the room will be shown
-                    in the published room list. A `private` visibility will hide
-                    the room from the published room list. Rooms default to
-                    `private` visibility if this key is not included. NB: This
-                    should not be confused with `join_rules` which also uses the
-                    word `public`.
+                    The room's visibility in the server's
+                    [published room directory](#published-room-directory).
```

This needs more anchoring, so that it works in https://playground.matrix.org/:

```suggestion
                    [published room directory](/client-server-api#published-room-directory).
```

---

Sorry, forgot about this again. 🤦‍♂️